hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -75,10 +75,14 @@ execute_robot_test(){
   CONTAINER="$1"
   TEST="$2"
   TEST_NAME=$(basename "$TEST")
-  TEST_NAME=${TEST_NAME%.*}
+  TEST_NAME="$(basename $COMPOSE_DIR)-${TEST_NAME%.*}"
   set +e
   OUTPUT_NAME="$COMPOSE_ENV_NAME-$TEST_NAME-$CONTAINER"
+  docker-compose -f "$COMPOSE_FILE" exec -T "$CONTAINER" mkdir -p "$RESULT_DIR_INSIDE"
   docker-compose -f "$COMPOSE_FILE" exec -e  SECURITY_ENABLED="${SECURITY_ENABLED}" -T "$CONTAINER" python -m robot --log NONE -N "$TEST_NAME" --report NONE "${OZONE_ROBOT_OPTS[@]}" --output "$RESULT_DIR_INSIDE/robot-$OUTPUT_NAME.xml" "$SMOKETEST_DIR_INSIDE/$TEST"
+
+  export FULL_CONTAINER_NAME=$(docker-compose -f "$COMPOSE_FILE" ps | grep "_${CONTAINER}_" | head -n 1 | awk '{print $1}')
+  docker cp $FULL_CONTAINER_NAME:$RESULT_DIR_INSIDE "$COMPOSE_DIR"
```

Comment:
shellcheck:13: note: Double quote to prevent globbing and word splitting. [SC2086]
shellcheck:34: note: Double quote to prevent globbing and word splitting. [SC2086]